src/main/java/application/controller/DeleteUserController.java

```diff
@@ -0,0 +1,24 @@
+package application.controller;
+
+import application.lib.Injector;
+import application.service.UserService;
+import java.io.IOException;
+import javax.servlet.ServletException;
+import javax.servlet.annotation.WebServlet;
+import javax.servlet.http.HttpServlet;
+import javax.servlet.http.HttpServletRequest;
+import javax.servlet.http.HttpServletResponse;
+
+@WebServlet("/delete")
```

Comment:
do we delete `user` or `product` or `shoppingCart`?

Reply:
done